src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -10206,6 +10206,15 @@ export class StudyViewPageStore
                     getSuffixOfMolecularProfile(this.defaultProtProfile)
                 );
             }
+            // Add RNA profile if no other profiles are selected
```

Comment:
@inodb why only do this when there are no other profiles selected?  it's arbitrary.  why not just add it either way, just like those above?